css/style.css

```diff
@@ -41,37 +40,41 @@ nav ul li {
   border-radius: 10% 10% 0 0;
 }
 
-/* CONTENT (ASIDE AND SECTION) LAYOUT */
-.content {
-  display: flex;
-  justify-content: space-between;
-  height: 500px;
-}
-
+/* ITEM SELECTION SECTION */
 .vote h2 {
   text-align: center;
   font-size: 1.5em;
+  border: 1px solid black;
+  padding: 10px;
+  height: 10%;
```

Comment:
Just as a note, percentage height generally works weirdly... which is all that I remember off the top of my head but [here's a StackOverflow with more details](https://stackoverflow.com/questions/1622027/percentage-height-html-5-css). Those details are that percentage height only works if you're setting the parent element's height explicitly.